### ISSUE TITLE
How to dump msdk log messages on Windows

### ISSUE THREAD

**Issue by tong1wu** [Intel-Media-SDK/MediaSDK#2830]
According to this issue, https://github.com/Intel-Media-SDK/MediaSDK/issues/948#issuecomment-442511043
I could dump MSDK log messages on Linux. But I don't have an idea where to create the /.mfx_trace on Windows. When I build the msdk source code with release mode there is nothing and for debug mode I could only see MSDK_HEVCE_HW_DDI_LOG.txt which is not what I want. 
It seems useless when I put the .mfx_trace under c:/Users. Is there any solution for Windows. 
And I also don't know which directory the log file will be put. Thanks.

**Comment by dmitryermilov** [comment]
It's much easier on Windows. MediaSDK dev packages have GUI tracer app. Please downloaded the latest dev package from official Intel website.

**Comment by tong1wu** [comment]
Thanks for the reply. I am using the open source oneVPL dispatcher and the inner-source mdp-msdk-lib to build the runtime. After I downloaded the tracer.log and pressed the start button, no bytes are written into the log file. Does the runtime that I'm using have to be the msdk runtime in the dev package? I put the libmfx64-gen.dll which I built myself under C:\Windows\System32 and seems that tracer.log doesn't react to it when I run the application to call the runtime.
